awx/ui_next/src/util/qs.js

```diff
@@ -14,6 +14,10 @@ export function getQSConfig(
   if (!namespace) {
     throw new Error('a QS namespace is required');
   }
+  // if order_by isn't passed, default to name
+  if (!Object.keys(defaultParams).filter(key => key === 'order_by').length) {
```

Comment:
Is there a reason you don't just check `if (!defaultParams.order_by)`?

Reply:
Good catch, much more direct way to get what I'm looking for hah